Code/GraphMol/Aromaticity.cpp

```diff
@@ -277,13 +277,13 @@ bool incidentCyclicMultipleBond(const Atom *at) {
 bool incidentMultipleBond(const Atom *at) {
   PRECONDITION(at, "bad atom");
   const auto &mol = at->getOwningMol();
-  int deg = at->getDegree() + at->getNumExplicitHs();
+  auto deg = at->getDegree() + at->getNumExplicitHs();
   for (const auto bond : mol.atomBonds(at)) {
     if (!std::lround(bond->getValenceContrib(at))) {
       --deg;
     }
   }
-  return at->getExplicitValence() != deg;
+  return at->getValence(true) != deg;
```

Comment:
I would prefer

```
const bool explicit = true;
return at->getValence(explicit);
```

However, getValence is used so much, this just might be something we need to just learn intrinsically.

Reply:
I'm not a big fan of using false and true for getValence due to my generally poor memory.  I would prefer an enum for clarity, but that ship might have sailed.

```
#include <iostream>
enum Valence : bool { Implicit = false, Explicit = true };

int getValence(Valence v) {
  if (v) return 1;
  else return 0;
}
int main (int ac, const char **av) {
  {
    auto a = getValence(Implicit);
    std::cout << (int) a << std::endl;
    auto v = getValence(Explicit);
    std::cout << (int) v << std::endl;
  }
  return 0;
}
```

Reply:
I think this is a good idea. I will make it so.